metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ghoneycutt-eyaml",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
The maintainer will update this upon release. For instance, I might integrate other work into the release or merge ofter PR's first and release them, so you would not want to make these types of changes for version numbers.